### PR TITLE
fix links

### PR DESCRIPTION
**Pull request by Fil**
after abef61b

### CODE REVIEW

**Review by mbostock — COMMENTED** [review]
Can you add a trailing slash to /inputs/ please.

File: docs/imports.md [hunk 1]
```diff
@@ -230,7 +230,7 @@ Click on any of the imported symbols below to learn more.
 <pre><code class="language-js">import * as <a href="./lib/duckdb">duckdb</a> from "npm:@duckdb/duckdb-wasm";</code></pre>
 <pre><code class="language-js">import {<a href="./lib/duckdb">DuckDBClient</a>} from "npm:@observablehq/duckdb";</code></pre>
 <pre><code class="language-js">import {<a href="./sql">sql</a>} from "npm:@observablehq/duckdb";</code></pre>
-<pre><code class="language-js">import * as <a href="./lib/inputs">Inputs</a> from "npm:@observablehq/inputs";</code></pre>
+<pre><code class="language-js">import * as <a href="./inputs">Inputs</a> from "npm:@observablehq/inputs";</code></pre>
```
Comment (Fil) [inline docs/imports.md:233]:
```suggestion
<pre><code class="language-js">import * as <a href="./inputs/">Inputs</a> from "npm:@observablehq/inputs";</code></pre>
```

File: docs/reactivity.md [hunk 2]
```diff
@@ -267,7 +267,7 @@ My favorite baseball team is the ${team}!
 My favorite baseball team is the ${team}!
 ```
 
-The above example uses `Inputs.radio`, which is provided by [Observable Inputs](./lib/inputs). You can also implement custom inputs using arbitrary HTML. For example, here is a [range input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) that lets you choose an integer between 1 and 15 (inclusive):
+The above example uses `Inputs.radio`, which is provided by [Observable Inputs](./inputs). You can also implement custom inputs using arbitrary HTML. For example, here is a [range input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) that lets you choose an integer between 1 and 15 (inclusive):
```
Comment (Fil) [inline docs/reactivity.md:270]:
```suggestion
The above example uses `Inputs.radio`, which is provided by [Observable Inputs](./inputs/). You can also implement custom inputs using arbitrary HTML. For example, here is a [range input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) that lets you choose an integer between 1 and 15 (inclusive):
```

**Review by Fil — COMMENTED** [review]
add a trailing slash

**Review by mbostock — APPROVED** [review]
We should have broken links break the build now.

**Pull request merged**